apps/tailwind-components/tests/vitest/composables/useForm.spec.ts

```diff
@@ -59,11 +65,20 @@ describe("useForm", () => {
         label: "columns 4",
         required: true,
       },
+      {
+        columnType: "BOOL",
+        id: "col6",
+        label: "columns 6",
+        required: true,
+      },
     ]);
   });
 
   test("should return a list of empty required fields", () => {
-    const formValues = ref<Record<string, columnValue>>({});
+    const formValues = ref<Record<string, columnValue>>({
+      // non empty required bool field
+      col6: false,
```

Comment:
could have used a meaningful id  instead of repeating this comments

Reply:
yea this reuses the setup for all the tests , maybe we should organize using describe or multiple test. not sure it's a bit of a trade off between reuses and specific test setup. might also be a bit of a code smell that we are doing to much in a single module